mrp_repair_widget_digitized_signature/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Alessio Gerace- Agile Business Group
+#
+#    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+#
+##############################################################################
```

Comment:
Please, follow the OCA model for the headings: https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py